test/framework/kubernetes.go

```diff
@@ -285,3 +289,73 @@ func checkPodContainerHasEnvVariableWithValue(pod *corev1.Pod, containerName, en
 	}
 	return false
 }
+
+// GetIngressURI returns the ingress URI
+func GetIngressURI(namespace, serviceName string) (string, error) {
+	ingress, err := WaitForIngress(namespace, serviceName)
+	if err != nil || ingress == nil || len(ingress.Spec.Rules) == 0 {
+		return "", fmt.Errorf("Ingress %s does not exist in namespace %s: %v", serviceName, namespace, err)
+	}
+
+	return fmt.Sprintf("http://%s:80", ingress.Spec.Rules[0].Host), nil
+}
+
+// WaitForIngress waits for ingress by a service
+func WaitForIngress(namespace, serviceName string) (*v1beta1.Ingress, error) {
```

Comment:
This function doesn't have much sense from my point of view.
It could be merged together with GetIngressURI() above

Reply:
Good idea. Merged.